single-function/lru-cache/submission.js

```diff
@@ -0,0 +1,66 @@
+class DoubleLinkedList {
+  constructor(key, value) {
+    this.key = key;
+    this.value = value;
+  }
+}
+
+class LRUCache {
+  constructor(capacity) {
+    this.capacity = capacity;
+    this.cache = {};
+    let head = new DoubleLinkedList(0, 0);
+    let tail = new DoubleLinkedList(0, 0);
+    head.next = tail;
+    tail.prev = head;
+    this.head = head;
+    this.tail = tail;
+    this.count = 0;
+  }
+
+  deleteDoubleLinkedList(node) {
+    node.prev.next = node.next;
+    node.next.prev = node.prev;
+  }
+
+  addNodeToHead(node) {
```

Comment:
From the function name i can't tell if you mean before or after the head.  Please change the name to be unambiguous about that.